runtime/dancebox/src/lib.rs

```diff
@@ -739,6 +741,53 @@ impl pallet_maintenance_mode::Config for Runtime {
 
 impl pallet_root_testing::Config for Runtime {}
 
+parameter_types! {
+    pub StakingAccount: AccountId32 = PalletId(*b"POOLSTAK").into_account_truncating();
+    pub const CurrencyHoldReason: [u8; 8] = *b"POOLSTAK";
+    pub const InitialManualClaimShareValue: u128 = currency::KILODANCE;
+    pub const InitialAutoCompoundingShareValue: u128 = currency::KILODANCE;
+    pub const InitialJoiningShareValue: u128 = currency::DANCE;
+    pub const InitialLeavingShareValue: u128 = currency::DANCE;
```

Comment:
The doc comment says

```
        /// Should be 1 to avoid rounding. (configurable for tests)
```

But currency::DANCE is not 1, it is 1_000_000_000_000.

```suggestion
    pub const InitialJoiningShareValue: u128 = 1;
    pub const InitialLeavingShareValue: u128 = 1;
```

Reply:
Those parameters were mostly added to be able to properly test rounding. If you see another way to customize the value in tests we could remove those type parameters.

Reply:
> If you see another way to customize the value in tests we could remove those type parameters.

Idk, maybe adding a storage item only for testing like we sometimes do for benchmarks, but for now you can leave it like this.